packages/manager/modules/pci/src/projects/project/storages/databases/database/databases/databases.html

```diff
@@ -0,0 +1,53 @@
+<div class="container-fluid px-0 mt-3">
+    <!-- error messages -->
+    <cui-message-container
+        data-messages="$ctrl.messages"
+    ></cui-message-container>
+
+    <h2 data-translate="pci_databases_databases_tab_title"></h2>
+
+    <oui-datagrid
+        class="mt-5"
+        data-page-size="10"
+        data-rows="$ctrl.databasesList"
+    >
+        <oui-datagrid-column
+            data-title=":: 'pci_databases_databases_tab_name' | translate"
+            data-property="name"
+            data-sortable="asc"
+            data-type="string"
+            data-searchable
+            data-filterable
+        ></oui-datagrid-column>
+        <oui-action-menu
+            data-compact
+            data-placement="end"
+            ng-if="!$row.default"
+        >
+            <oui-action-menu-item data-on-click="$ctrl.deleteDatabase($row)">
+                <span
+                    data-translate="pci_databases_databases_tab_delete_link"
+                ></span>
+            </oui-action-menu-item>
+        </oui-action-menu>
+        <oui-datagrid-topbar>
+            <oui-button data-on-click="$ctrl.addDatabase()">
+                <i class="oui-icon oui-icon-add pr-1" aria-hidden="true"></i>
+                <span data-translate="pci_databases_databases_tab_add"></span>
+            </oui-button>
+            <button
+                data-ng-if="$ctrl.databasesList.length > 0"
+                class="float-right oui-button oui-button_secondary oui-button_icon-only oui-button_s"
+                type="button"
+                data-title="{{:: 'pci_databases_common_refresh' | translate }}"
+                data-ng-click="$ctrl.refreshing=true; $ctrl.refreshDatabases()"
```

Comment:
Can we move `$ctrl.refreshing=true;` into the `refreshDatabases` method?

Reply:
Done here: https://github.com/ovh/manager/pull/5488/commits/45d16da89ff2c70503a9a111c3fd5996a5eecbe1